tests/drawer/test_tape_mpl.py

```diff
@@ -974,11 +983,11 @@ def test_single_mcm_measure(self):
 
         # probably not a good way to test this, but the best I can figure out
         assert pe1._gc == {
-            "linewidth": 5 * plt.rcParams["lines.linewidth"],
+            "linewidth": 5 * 1.5,  # lines.linewidth for black white style
             "foreground": "black",  # lines.color for black white style
         }
         assert pe2._gc == {
-            "linewidth": 3 * plt.rcParams["lines.linewidth"],
+            "linewidth": 3 * 1.5,  # lines.linewidth for black white style
```

Comment:
Why was this changed? Seems like better testing practice to me to not use hard-coded values.

Reply:
My latest push fixed those issues. I reverted the changes 👌